packages/fxa-payments-server/src/store/reducers.ts

```diff
@@ -54,21 +71,21 @@ export const reducers = {
       ),
       [updatePayment.toString()]: fetchReducer('updatePayment'),
       [resetCreateSubscription.toString()]: setStatic({
-        createSubscription: fetchDefault(null),
+        createSubscription: defaultState.api.createSubscription,
```

Comment:
We can also re-use values from `defaultState` in these reset actions, rather than re-creating them